components/layout/grid/src/_settings.scss

```diff
@@ -0,0 +1,67 @@
+$lg-breakpoints: $breakpoints !default;
```

Comment:
I think we should have only one breakpoint source of truth and that should be $breakpoints, is having different breakpoint definition inside a business unit a real use case?

Reply:
Agree!

I will move your question to the SUI Meeting to know how the rest of verticals work.